docs/docs/recipes.md

```diff
@@ -2,7 +2,102 @@
 title: Recipes
 ---
 
-This is a stub. Help our community expand it.
+<!-- Basic template:
+Task to accomplish.
+1-2 sentences about it.
+Relevant links out (tutorial, doc pages, plugin readmes, etc).
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+Links:
+* tutorial link
+* docs link
+* additional, if needed
+
+And yeah — those three things are exactly what we're thinking. A first step would be to just go through the tutorial and pull out all the basic things we teach there in a condensed form e.g. creating a site, creating a page, linking between pages, etc. -->
+
+Craving a happy medium between doing the [full tutorial](/tutorial/) and crawling the [full docs]((/tutorial/))? Here's a quick guiding reference for how to build things, Gatsby style.
+
+## Table of Contents
+- [Using a starter](#using-a-starter)
+- [Creating pages](#creating-pages)
+- [Linking between pages](#linking-between-pages)
+- [Styling](#styling)
+- [Creating layouts](#creating-layouts)
+- [Deploying](#deploying)
+- [Querying data](#querying-data)
+- [Sourcing data](#sourcing-data)
+- [Transforming data](#transforming-data)
+
+## Using a starter
+
+Starters are boilerplate Gatsby sites maintained officially, or by the community.
+
+* Learn how to use the Gatsby CLI tool to use starters in [tutorial part one](/tutorial/part-one/#using-gatsby-starters)
+* See a list of [official and community starters](/docs/gatsby-starters/)
+* Check out Gatsby's [official default starter](https://github.com/gatsbyjs/gatsby-starter-default)
+
+## Creating pages
+
+You can create pages in Gatsby explicitly by definining React components in `src/pages/`, or programmatically by using the `createPages` API. 
+
+* Walk through creating a page by defining a React component in `src/pages` in [tutorial part one](/tutorial/part-one/#familiarizing-with-gatsby-pages)
```

Comment:
One thing I'd like to do is make sure all the tutorial information exists in the Docs in guide form so that we guarantee each doc is fairly independent and requires as little context as possible, which would mean these links will change eventually. For the sake of saving time, I think it makes sense to link to the tutorial right now since that's where the information lives. What say ye @KyleAMathews?

Reply:
Yeah I looked at this as a context-gathering mission, centralizing related topics addressed in different places at different levels. Everything in this doc that links out to the tutorial is led into with "Walk through", indicating it's more supportive. I agree in theory but yeah, right now just centralizing the different pieces that might be useful to people, to minimize the searching.